### PR TITLE
[ICDS Dashboard] pull site code for dashboard

### PR DESCRIPTION
**Pull request by Rohit25negi**
currently for the site code location type is coming. This code fixes that
ticket: https://dimagi-dev.atlassian.net/browse/ICDS-869

### CODE REVIEW

File: custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py [hunk 1]
```diff
@@ -301,5 +301,5 @@ def _get_all_locations_for_domain(domain):
     return (
         SQLLocation.objects
         .filter(domain=domain, is_archived=False)
-        .values('pk', 'parent_id', 'metadata', 'name', 'location_id', 'location_type__code')
+        .values('pk', 'parent_id', 'metadata', 'name', 'location_id', 'location_type__code', 'site_code')
```
Comment (snopoke) [inline custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py]: is `location_type__code` still needed?

File: custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py [hunk 2]
```diff
@@ -359,5 +359,5 @@ def _get_all_locations_for_domain(domain):
     return (
         SQLLocation.objects
         .filter(domain=domain, is_archived=False)
-        .values('pk', 'parent_id', 'metadata', 'name', 'location_id', 'location_type__code')
+        .values('pk', 'parent_id', 'metadata', 'name', 'location_id', 'location_type__code', 'site_code')
```
Comment (snopoke) [inline custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py]: same question as above

**Review by snopoke — APPROVED** [review]

File: custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py]: yeah. Its being used above in the code.

File: custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py (hunk 2 above)
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py]: yeah.

**Comment by calellowitz** [comment]
Is the current site code used at all (the one thats really just the name of the location type)?

**Review by emord — APPROVED** [review]

**Comment by Rohit25negi** [comment]
> Is the current site code used at all (the one thats really just the name of the location type)?

Not that I can think of.

**Pull request merged**